src/utils.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module wordcount/utils
+ */
+
+/**
+ * Function walks through all the model's nodes. It obtains a plain text from each {@link module:engine/model/text~Text}
```

Comment:
Description of implementation ;)

Something like: "Returns plain text representation of an element and it's children. The blocks are separated by a newline (`\n` ).